### DISCUSSION TITLE
foundry install

### DISCUSSION THREAD

**Discussion by cell0901** [Cyfrin/foundry-full-course-cu#977] [Q&A]
getting this error while installing foundry
```
curl: (28) Failed to connect to raw.githubusercontent.com port 443 after 130798 ms: Connection timed out
```

**Comment by theirrationalone** [comment]
foundry has no support for windows...,

If you're using linux or variant i.e., ubuntu or etc.... then please try to use some `VPN` while installing foundry.

**Reply by cell0901** [reply]
Which VPN is free, can u recommend any?

**Reply by cell0901** [reply]
i am using vpn book, but its still not downloading, stuck on this
![Uploading image.png…]()

**Reply by theirrationalone** [reply]
Please upload image again with this syntax \[image_name\]\(image_link\), Therefore it is broken.

**Reply by theirrationalone** [reply]
Please refer this [Read wisely](https://github.com/Cyfrin/foundry-full-course-f23/discussions/808#discussion-5597726), If you found any solution ;`)

**Reply by Kanishka227** [reply]
![Screenshot (31)](https://github.com/Cyfrin/path-solidity-developer-2023/assets/123467333/f95179fc-de77-42fb-89e8-ce7510de4e78)

Still unable to install . Can anyone help

**Reply by cell0901** [reply]
Try vpn, this website site doesn't work in India. I tried vpn and worked

**Reply by Kanishka227** [reply]
Yes it worked after using vpn . Thanks.

**Comment by olahamid** [comment]
> getting this error while installing foundry
> 
> ```
> curl: (28) Failed to connect to raw.githubusercontent.com port 443 after 130798 ms: Connection timed out
> ```

you could try using gitpod as well

**Comment by alfheimrShiven** [comment]
@cell0901 I had solved this issue by switching my DNS to Google's Public DNS `8.8.4.4`

Change the DNS of your system to `8.8.4.4` and try again. Do let me know if it works!

**Comment by Kanishka227** [comment]
> > getting this error while installing foundry
> > ```
> > curl: (28) Failed to connect to raw.githubusercontent.com port 443 after 130798 ms: Connection timed out
> > ```
> 
> you could try using gitpod as well

**Comment by sagoto80** [comment]
can anyone help on this https://github.com/Cyfrin/foundry-full-course-f23/discussions/1181
